addons/mail/models/mail_thread.py

```diff
@@ -2199,7 +2199,7 @@ def message_notify(self, *,
         self._notify_thread(new_message, msg_values, **notif_kwargs)
         return new_message
 
-    def _message_log_with_view(self, views_or_xmlid, **kwargs):
```

Comment:
@kgasior, why did you change the method signature? :thinking: 

First of all, makes sense that helper method is private, so, starts with` _`, but more importantly, we can't change method signature in stable. For more info - https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean 

So, change it back :smile: